web/accounts/tests/test_models.py

```
@@ -0,0 +1,62 @@
+import pytest
+from django.contrib.auth import get_user_model
+
+pytestmark = pytest.mark.django_db
+
+
+def test_user_uses_email_as_username():
+    User = get_user_model()
+    user = User.objects.create_user(email="alice@example.com", password="pw12345678")
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "password"

(S106)